lms/static/js/spec/learner_dashboard/course_card_view_spec.js

```diff
@@ -0,0 +1,78 @@
+define([
+        'backbone',
+        'jquery',
+        'js/learner_dashboard/views/course_card_view'
+    ], function (Backbone, $, CourseCardView) {
+        
+        'use strict';
+        
+        describe('Course Card View', function () {
+            var view = null,
+                courseCardModel,
+                setupView,
+                context = {      
+                    certificate_url: '',
+                    course_end: 'Jun 13, 2016',
+                    course_modes: [],
+                    course_key: 'course-v1:ANUx+ANU-ASTRO1x+3T2015',
+                    courseware_url: 'http://localhost:8000/courses/course-v1:edX+DemoX+Demo_Course/info',
+                    course_start: 'Apr 25, 2016',
+                    course_started: true,
+                    display_name: 'Astrophysics: Exploring Exoplanets',
+                    image_url: 'https://testimage.com/image',
+                    key: 'ANU-ASTRO1x',
+                    marketing_url: 'https://www.edx.org/course/astrophysics-exploring-exoplanets-anux-anu-astro2x-1',
+                    organization: {
+                        display_name: 'Australian National University',
+                        key: 'ANUx'
+                    },
+                    run_modes: [{
+                        course_key: '12313',
+                        mode_slug: 'verified',
+                        run_key: '2T2016',
+                        start_date: ''
+                    }]
+                };
+
+            setupView = function(enrollment_status){
+                context.enrollment_status = enrollment_status;
+                setFixtures('<div class="course-card card"></div>');
+                courseCardModel = new Backbone.Model(context);
+                view = new CourseCardView({
+                    model: courseCardModel
+                });
+            };
+
+            beforeEach(function() {
+                setupView(null);
+            });
+
+            afterEach(function() {
+                view.remove();
+            });
+
+            it('should exist', function() {
+                expect(view).toBeDefined();
+            });
+
+            it('should render the course card based on the data enrolled', function() {
+                view.remove();
+                setupView('enrolled');
+                expect(view.$('.header-img').attr('src')).toEqual(context.image_url);
+                expect(view.$('.course-details h3').html()).toEqual(context.display_name);
+                expect(view.$('.course-details p').html()).toEqual(context.key);
+                expect(view.$('.course-details span').html())
```

Comment:
I think the elements should have classes which we use for the tests, especially with the need to update this code later for styles. I don't want the tests to fail is we decide next week that the start and end dates should be in a div instead of a span.